tools/install_travis.sh

```diff
@@ -50,5 +50,8 @@ pip install -r companion.txt
 echo -e ">> [`date`] installing mpi4py"
```

Comment:
Are we using this script anymore?

Reply:
Hmmm - that might be why Im having trouble getting the CI to work! I'll have a look once i am back from holiday